Gemfile.lock

```diff
@@ -45,8 +45,14 @@ GEM
     addressable (2.5.2)
       public_suffix (>= 2.0.2, < 4.0)
     arel (8.0.0)
+    autoprefixer-rails (8.2.0)
+      execjs
     bcrypt (3.1.11)
     bindex (0.5.0)
+    bootstrap (4.0.0)
+      autoprefixer-rails (>= 6.0.3)
+      popper_js (>= 1.12.9, < 2)
+      sass (>= 3.5.2)
```

Comment:
oh interesting - I have been adding gem 'bootstrap-sass' to my gemfile, but it looks like it installs that anyways 👍

Reply:
I think..